vehicle/mkvi/software/bms/bms_config.c

```diff
@@ -49,10 +49,13 @@ timer_cfg_s timer1_cfg = {
     .timer1_mode = TIMER1_MODE_PHASE_CORRECT_PWM_10_BIT,
     .prescalar = CLKIO_DIV_8,
     .channel_a = {
-        .output_compare_match = 0x50,
+        .output_compare_match = 12,
```

Comment:
would be nice to include documentation on where the compare match values come from